src/nn/nn.c

```diff
@@ -161,6 +162,9 @@ void nn_set_wlen(nn_t A, u8 new_wlen)
 	u8 i;
 	if (A->wlen == new_wlen) {
 		return;
+	} else if (A->wlen < new_wlen) {
+		A->wlen = new_wlen;
+		return;
```

Comment:
Try removing this and test the performance changes. This change might be dangerous.

Reply:
The cycles consumed before reverting commit https://github.com/cryptape/libecc/pull/1/commits/b6538a012a80b9c9b1acc302a4950df42367a527
```
ckb-debugger --pprof=build/secp256r1_bench.pprof --bin build/secp256r1_bench
start main...
signature verification succeeded

Run result: 0
Total cycles consumed: 7509738(7.2M)
Transfer cycles: 15706(15.3K), running cycles: 7494032(7.1M)
```

The cycles consumed after reverting commit https://github.com/cryptape/libecc/pull/1/commits/b6538a012a80b9c9b1acc302a4950df42367a527
```
ckb-debugger --pprof=build/secp256r1_bench.pprof --bin build/secp256r1_bench
start main...
signature verification succeeded

Run result: 0
Total cycles consumed: 7569149(7.2M)
Transfer cycles: 15702(15.3K), running cycles: 7553447(7.2M)
```